### PR TITLE
Provide better LABELs for summary and description

### PR DESCRIPTION
**Pull request by hhorak**

### CODE REVIEW

**Comment by hhorak** [comment]
@torsava PTAL

File: 2.7/Dockerfile.rhel7 [hunk 1]
```diff
@@ -11,16 +11,28 @@ ENV PYTHON_VERSION=2.7 \
...
+      io.k8s.description="$SUMMARY" \
+      io.k8s.display-name="Python 2.7" \
+      io.openshift.expose-services="8080:http" \
+      io.openshift.tags="builder,python,python27,rh-python27"
+
 # Labels consumed by Red Hat build service.
 LABEL com.redhat.component="python27-docker" \
       name="rhscl/python-27-rhel7" \
       version="2.7" \
       release="1" \
```
Comment (torsava) [inline 2.7/Dockerfile.rhel7]: I've looked through other s2i containers and some are permanently on release="1" while others have higher ones. What constitutes a reason to do a release bump in containers?

**Comment by torsava** [comment]
I sent you a PR to your PR with the mentioned changes.

File: 2.7/Dockerfile.rhel7 (hunk 1 above)
Comment (pkubatrh) [inline 2.7/Dockerfile.rhel7]: >What constitutes a reason to do a release bump in containers?
afaik we do not do a release bump in upstream Dockerfiles.
The existing release values > 1 in other containers are probably changes that got upstream from rhel images, where we need to do proper versioning (major release bump for devel->beta, beta->GA and minor release bump for rebuilds).

**Comment by torsava** [comment]
Changes look good, tests are passing—merging.

**Pull request merged**